Small logic improvements

newnon opened the pull request:
Added 4 state parameters
background color
background opacity
title color
title opacity

newnon committed:
buttons cascade color & cascade opacity fix

minggo commented:
@newnon
Could you please fix conflict?

@andyque
Could you please take a look?

newnon commented:
Done

minggo commented:
Thanks @newnon .

minggo commented on tools/cocos2d-console:
```diff
@@ -1 +1 @@
-Subproject commit b49fad3fb2dbc396e71a5b1178cf2532309cd174
+Subproject commit fed86b061f5bacffa66149498afdcba180bd0063
```
Please don't modify submodules.

newnon commented:
Sorry it's merge bug

newnon committed:
Small logic improvements
Added 4 state parameters
background color
backroudnd opacity
title color
title opacity

newnon commented:
everything clear

minggo commented:
Thanks.
